src/include/miopen/solver.hpp

```diff
@@ -1981,6 +1981,83 @@ struct ConvHipImplicitGemmWrwV4R4Xdlops : SolverBase<ConvolutionContext>
     PerformanceImplicitGemmWrwV4R4Xdlops Search(const ConvolutionContext&,
                                                 const AnyInvokeParams& invoke_ctx) const;
 };
+
+struct PerformanceImplicitGemmWrwV4R4Xdlops_Padded_Gemm
+    : Serializable<PerformanceImplicitGemmWrwV4R4Xdlops_Padded_Gemm>
+{
+    int GemmMPerBlock;
+    int GemmNPerBlock;
+    int GemmKPerBlock;
+    int GemmMPerWave;
+    int GemmNPerWave;
+    int GemmKPack;
+    int GemmMFactor;
+    int GemmNFactor;
+    int GemmKFactor;
+    bool GemmAThreadCopyMoreGemmK;
+    bool GemmBThreadCopyMoreGemmKPack;
```

Comment:
change it to GemmBThreadCopyMoreGemmK

Reply:
@asleepzzz Please check the non-padded bwd-wrw kernel, this part of code has been updated in that kernel.

Reply:
if(GemmBThreadCopyMoreGemmKPack)
        {
            data_per_thread_copy_gemmk = gcd(GemmKPerBlock, tmp);
            data_per_thread_copy_gemmn = tmp / data_per_thread_copy_gemmk;
        }
        else
        {
            data_per_thread_copy_gemmn = gcd(GemmKPerBlock, tmp);
            data_per_thread_copy_gemmk = tmp / data_per_thread_copy_gemmn;
        }
non-padding kernel should also change name